services/github.py

```diff
@@ -0,0 +1,15 @@
+import subprocess
+
+from services.exceptions import GitHubException
+
+
+class GitHubService:
```

Comment:
Nice, I dig it. Found a library to consider if we need to hit it more programmatically in the future (https://pygithub.readthedocs.io/en/stable/introduction.html).